### ISSUE TITLE
Can not import react module into the app

### ISSUE THREAD

**Issue by Th3F0x-code** [porscheofficial/cookie-consent-banner#21]
**Node/npm version(s)**: v18.16.1

**Package(s) version(s)**: ^3.1.3

**Description of the issue/feature**:  The package cannot be imported into the app, which always returns the same error (I have also tried with the new version but nothing changes)

package.json
```
    {
  "name": "react_",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@porscheofficial/cookie-consent-banner-react": "^3.1.3",
    "@statuspage/status-widget": "^1.0.4",
    "@stripe/react-stripe-js": "^2.1.0",
    "@stripe/stripe-js": "^1.54.1",
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^14.0.0",
    "@testing-library/user-event": "^14.4.3",
    "aos": "^2.3.4",
    "firebase": "^8.6.8",
    "history": "^5.3.0",
    "i18next": "^23.2.11",
    "i18next-browser-languagedetector": "^7.1.0",
    "i18next-http-backend": "^2.2.1",
    "lottie-react": "^2.4.0",
    "lottie-web": "^5.12.2",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-ga4": "^2.1.0",
    "react-i18next": "^13.0.2",
    "react-owl-carousel": "^2.3.3",
    "react-router-dom": "^6.4.3",
    "react-router-hash-link": "^2.4.3",
    "react-scripts": "^5.0.1",
    "react-scrollspy": "^3.4.3",
    "shebang-regex": "^4.0.0",
    "web-vitals": "^2.1.4"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "@babel/plugin-proposal-private-property-in-object": "^7.21.11"
  }
}

```

***Error***
```
ERROR in ./src/component/GDPR_Banner/Main.js 4:0-118
Module not found: Error: Can't resolve '@porscheofficial/cookie-consent-banner-react@^3.1.3' in '/Volumes/SSD/shoppup_site/client/src/component/GDPR_Banner'
```

***Import Statement***
```
import {CookieConsentBanner, triggerCookieConsentBanner} from '@porscheofficial/cookie-consent-banner-react';
```

**Comment by strandchef** [comment]
I have got the same issue.

**Comment by pboeder** [comment]
Thanks for reporting @Th3F0x-code and @strandchef.
Can you confirm that this issue didn't happen in earlier versions (<=3.1.3)?

**Comment by Th3F0x-code** [comment]
@pboeder yes exactly

**Comment by pboeder** [comment]
Should be fixed with `v3.1.5`. Sorry for the inconvenience!
